src/resources/css/Common.module.css

```diff
@@ -1,4 +1,4 @@
-@import '@digdir/design-system-tokens/brand/digdir/tokens.css';
+@import '@digdir/design-system-tokens/brand/altinn/tokens.css';
 
 body {
   margin: 0;
```

Comment:
Unchanged files below have lint errors and I'm trying to get rid of these (they get merged into main when some branch based on a branch that has them gets merged into main). Try to fix those warnings in below files manually or run yarn lint --fix

Reply:
Done 😊